wordpress-mobile/WordPress-Android
# Onboarding Improvements: Enable Feature Flag

Parent #14989

This PR enables the feature flag for the `Onboarding Improvements: Existing Users` feature.

-----

To test:
1. Clear app data to make sure all feature flags are set to default.
2. Log-out and log-in to the app.
3. Make sure the new `Login Epilogue` screen is shown.
    - Select a site from the list of sites and make sure the new `Quick Start Prompt` is shown. Also, verify that clicking `Show me around`, or `No thanks` or dismissing the screen, all flows are working as expected.
    - Click on the `Create New Site` button and make sure the `Choose a design` site creation screen is shown.  Also, verify that going through the site creation screen, all flows are working as expected, included the fact that the new `Quick Start Prompt` will be shown at the end.
4. When within the `My Site` screen, having completed the login flow, go to the `Site Picker` screen and create a new `WordPress.com` site.
5. Make sure the `Choose a design` site creation screen is shown and verify that going through the site creation screen, all flows are working as expected, included the fact that the new `Quick Start Prompt` will be shown at the end.

-----

## Regression Notes
1. Potential unintended areas of impact

N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)

See `To test` section above.

3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
